test/unit/mocks/entities.js

```diff
@@ -183,6 +183,12 @@ const organizationMembershipMock = {
   })
 }
 
+const organizationSpaceMembershipMock = {
```

Comment:
You can remove this and use `spaceMembershipMock` instead.
Also it seems that they are missing some properties like `admin` and `roles`. Don't know how important they are in the tests though.

Reply:
@guilebarbosa sure, on my way 🚐